packages/terminology/src/router.ts

```diff
@@ -0,0 +1,67 @@
+import { Router, Request, Response } from "express";
+import { codeSystemImportHandler } from "./operations/codeImport";
+import { codeSystemLookupHandler } from "./operations/codeLookup";
+import { conceptMapImportHandler } from "./operations/conceptMapImport";
+import { conceptMapTranslateHandler } from "./operations/conceptMapTranslate";
+import { FhirRequest, HttpMethod } from "@medplum/fhir-router";
+import { asyncHandler } from "./util";
+
+const fhirRouter = Router();
+
+function createFhirRequest(req: Request): FhirRequest {
```

Comment:
"create" gives the impression we're creating a request to be issued, while this is parsing the existing request (could also say converting from HTTP to FHIR request). Can we rename this to one of those, please?